packages/api/app/Models/Role.js

```diff
@@ -17,14 +16,13 @@ class Role extends Model {
 	}
 
 	static async getRole(role) {
-		const userRole = await this.query()
-			.where({ role: role.toUpperCase() })
-			.first();
-
-		if (!userRole) {
-			throw CE.ModelNotFoundException.raise('Role');
+		const userRole = this.query();
+		try {
+			userRole.where({ role: role.toUpperCase() });
+		} catch (error) {
+			userRole.where({ id: role });
```

Comment:
Esse try catch está funcionando? O propósito é colocar o where pelo id após dar erro tentando fazer um `toUpperCase()` num Number. É isso mesmo? Se for, não está nada claro que é esse o funcionamento esperado 😅
Caso tenha achado esquisita a forma que sugeri pra saber se é um id ou não, pode procurar outras formas de checar, porém, esta não me parece uma boa 🤔

Reply:
Entendi. Já apliquei a sugestão 👍